### DISCUSSION TITLE
develop without htmlPlugin

### DISCUSSION THREAD

**Discussion by fnicastri** [web-infra-dev/rsbuild#2726] [General]
How I can load the bundle without using the **htmlPlugin**?

I'm developing some components (not webcomponents) that I need to include in a larger
dotnet web app.
I have two entrypoints, a svelte component and a plain js/ts part (the svelte svtore that is shared with the rest of the system)

How I can configure rsbuild to emit the bundles in a way that I can load them from the web app during development?
Currently I load the files from Vite dev server but I have issues debugging them (known vite issue with svelte)

thanks

**Comment by chenjiahan** [comment]
How about disabling the html plugin and using the manifest file to load assets:

- https://rsbuild.dev/config/tools/html-plugin#disable-html
- https://rsbuild.dev/config/output/manifest

**Reply by fnicastri** [reply]
> How about disabling the html plugin and using the manifest file to load assets:
> 
> * https://rsbuild.dev/config/tools/html-plugin#disable-html
> * https://rsbuild.dev/config/output/manifest

Am I mistaken or the manifest.json is generated only during the build?
I have issues accessing it in dev mode.

**Reply by fnicastri** [reply]
I managed to activate the manifest (maybe it was some misconfiguration)
and loaded it in the index.html with `<link rel="manifest" href="/manifest.json" />`
but can't make the app start.

Can you point me to a working example?

Thank you
